fastly/notifications.go

```diff
@@ -0,0 +1,311 @@
+package fastly
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+// Integration holds the configuration for one integration.
+type Integration struct {
+	CreatedAt   *time.Time         `json:"created_at"`
+	Description *string            `json:"description"`
+	Config      *map[string]string `json:"config"`
```

Comment:
We don't need the `*` for a `map` or slice type (e.g. `*[]Integration` defined below) as those types are already pointers (i.e. if you don't set a value for the field, then it will be `nil`).

Example:
https://play.golang.com/p/CddfmQajQ-N